server/push/fcm/push_fcm.go

```diff
@@ -37,15 +37,112 @@ type Handler struct {
 	client *fcm.Client
 }
 
+// Configuration of AndroidNotification payload.
+type androidConfig struct {
```

Comment:
Can you move it closer to other struct definitions for better readability please?

Reply:
The logic I followed was type definition immediately followed by all the methods of the type, then the next type definition. I can do [type def 1], [type def 2]..., [methods for type 1], etc but I don't think it's objectively better, just a matter of preference.